aws/resource_aws_acm_certificate.go

```diff
@@ -378,6 +409,29 @@ func convertValidationOptions(certificate *acm.CertificateDetail) ([]map[string]
 	return domainValidationResult, emailValidationResult, nil
 }
 
+// sortDomainValidationOptions sorts a slice of domain validation options
+// according to the order of domain names in the subject alternatnive names slice
```

Comment:
"alternatnive" -> "alternative"

Reply:
I must have missed a typo from the source patch. Already fixed a few of those. Will update. Thanks!